pkg/token/cache.go

```diff
@@ -132,3 +140,50 @@ func (c *LockedTokenCache) Clear() {
 	}
 	c.memoryUsage = 0
 }
+
+var (
+	tokenExpiresInMetric = "token_expires_in_seconds"
+	tokenExpiresInHelp   = "Indicates remaining time until the token expires."
```

Comment:
```suggestion
	tokenExpiresInHelp   = "Indicates remaining time until the token expires"
```

Reply:
@WindzCUHK fixed in https://github.com/AthenZ/k8s-athenz-sia/pull/77/commits/732ace5f79e2f591ea5d7a96bd43c702a934453c